PG_VERSION no longer found

`make docker-build` (and hence, TravisCI builds) is/are failing in postgres on master due to:

```
E: Version '9.4.7-1.pgdg80+1' for 'postgresql-9.4' was not found
E: Version '9.4.7-1.pgdg80+1' for 'postgresql-contrib-9.4' was not found
The command '/bin/sh -c apt-get update  && apt-get install -y postgresql-common     && sed -ri 's/#(create_main_cluster) .*$/\1 = false/' /etc/postgresql-common/createcluster.conf     && apt-get install -y       postgresql-$PG_MAJOR=$PG_VERSION        postgresql-contrib-$PG_MAJOR=$PG_VERSION    && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
make: *** [docker-build] Error 1
```

We need to find the next acceptable `PG_VERSION` that can actually be found/used.

--- a/rootfs/Dockerfile
+++ b/rootfs/Dockerfile
@@ -21,7 +21,7 @@ ENV LANG en_US.utf8
 RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
 
 ENV PG_MAJOR 9.4
-ENV PG_VERSION 9.4.7-1.pgdg80+1
+ENV PG_VERSION 9.4.8-1.pgdg80+1
 
 RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main' $PG_MAJOR > /etc/apt/sources.list.d/pgdg.list
 

